mirrord-layer/src/lib.rs

```diff
@@ -119,19 +115,16 @@ async fn handle_hook_message(
                 service,
                 hints,
             });
-            let codec_result = codec.send(request).await;
-
-            trace!("HookMessage::GetAddrInfo codec_result {:#?}", codec_result);
+            let _codec_result = codec.send(request).await;
```

Comment:
why not `?` it

Reply:
This would change behavior, as we're currently just "eating" the error (note how these functions don't return `Result`).

Should I go for it (refactor `handle_x` to deal with these `codec.send` errors)?